src/RealND.hpp

```diff
@@ -135,7 +137,7 @@ namespace espressopp {
       data[i] = v[i];
   }
   
-  inline RealND::RealND(const int _dim, const real v[3]) {
+  inline RealND::RealND(const int _dim, const real v[]) {
     setDimension(_dim);
```

Comment:
I pushed simple tests for the constructors of RealND. Can put more test cases for the various methods; right now just for the constructors since the fix was in one of the constructor. We can talk about.